> Testing this branch in a custom job not to upload anything: https://build.osrfoundation.org/job/_test_ign-physics2-debbuilder/

I tried to build 2.1.0 with this branch and the software is not installing `engine-*` directories. I have seen this problem before but never explore the root cause of it:

```
dh_install: warning: Cannot find (any matches for) "usr/lib/*/ign-physics-[0-99]/engines/libignition-physics*-tpe-plugin.so" (tried in ., debian/tmp)

dh_install: warning: libignition-physics2-tpe-dev missing files: usr/lib/*/ign-physics-[0-99]/engines/libignition-physics*-tpe-plugin.so
dh_install: warning: Cannot find (any matches for) "usr/lib/*/ign-physics-[0-99]/engine-plugins/libignition-physics*-tpe-plugin*.so.*" (tried in ., debian/tmp)

dh_install: warning: libignition-physics2-tpe missing files: usr/lib/*/ign-physics-[0-99]/engine-plugins/libignition-physics*-tpe-plugin*.so.*
dh_install: error: missing files, aborting
make[1]: *** [debian/rules:15: override_dh_install] Error 2
```

--- a/ubuntu/debian/libignition-physics-tpe-dev.install
+++ b/ubuntu/debian/libignition-physics-tpe-dev.install
@@ -1,5 +1,5 @@
 usr/include/ignition/physics*/ignition/physics/tpe-plugin/*
 usr/lib/*/cmake/ignition-physics[0-99]-tpe-plugin/*
 usr/lib/*/libignition-physics[0-99]-tpe-plugin.so
-usr/lib/*/ign-physics-[0-99]/engines/libignition-physics*-tpe-plugin.so
+usr/lib/*/ign-physics-[0-99]/engine-plugins/libignition-physics*-tpe-plugin.so
 usr/lib/*/pkgconfig/ignition-physics[0-99]-tpe-plugin.pc

